charts/apps/kubeflow-pipelines/rds-s3-static/templates/ConfigMap/ml-pipeline-ui-configmap-kubeflow-ConfigMap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+data:
+  viewer-pod-template.json: "{\n  \"spec\": {\n    \"containers\": [\n      {\n  \
```

Comment:
maybe reformat this by grabbing a more readable yaml file from `tools/helmify/generated_output/kustomized_output_files`. This is due to the python yaml write library

Reply:
will- do when I regenearte with the alpha 7 version